encrypted-media-respec.html

```diff
@@ -2115,13 +2139,12 @@
                   <p>
                     If this object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
                   </p>
+                  <p class="note">
+                    This step may be run asynchronously, including after the document has unloaded.
```

Comment:
We need to normatively specify that steps are asynchronously only when if affects observable behavior. In a method definition, we need to specify which steps occur before and which occur after the method returns. Outside a method invocation, I don't think there is any implication that steps are blocking the main JS thread (if there's no method invokation, then _where_ is there thread blocked in the Javascript code ?).

So, here we are really just noting the fact that there are no observable consequences if these steps are executed later and thus implementations are free to execute them later, similar to `<a ping>` and beacon.